assets/javascript/app.js

```diff
@@ -0,0 +1,19 @@
+window.onload = function() {
+
+};
+
+
+
+var intervalId;
+var clockRunning = false;
+
```

Comment:
The start function is not being called at the beginning which in turn is not running the clock.